R/theme_gtsummary_roche.R

```diff
@@ -51,6 +51,7 @@ theme_gtsummary_roche <- function(set_theme = TRUE) {
       list(
         "tbl_summary-fn:percent_fun" = gtsummary::label_style_number(digits = 1, scale = 100),
         "pkgwide-fn:pvalue_fun" = label_roche_pvalue(),
+        "pkgwide-fn:pvalue_fun" = label_roche_percent(),
```

Comment:
This is supposed to be `"tbl_summary-fn:percent_fun" = label_roche_percent(),`

Reply:
updated :)